addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -103,34 +103,54 @@ $h4-font-size: o-website-value('h4-font-size') or $font-size-base * $o-theme-h4-
 $h5-font-size: o-website-value('h5-font-size') or $font-size-base * $o-theme-h5-font-size-multiplier !default;
 $h6-font-size: o-website-value('h6-font-size') or $font-size-base * $o-theme-h6-font-size-multiplier !default;
 
+$display-font-sizes: o-website-value('display-font-sizes') !default;
+
+$small-font-size: o-website-value('small-font-size') !default;
+
 // H2~H6 font families are custom variables.
 $headings-font-family: $o-theme-headings-font !default;
 $h2-font-family: $o-theme-h2-font !default;
 $h3-font-family: $o-theme-h3-font !default;
 $h4-font-family: $o-theme-h4-font !default;
 $h5-font-family: $o-theme-h5-font !default;
 $h6-font-family: $o-theme-h6-font !default;
+$display-1-font-family: $o-theme-display-1-font !default;
```

Comment:
> I find a bit confusing that we define custom variables in this file, especially because are not prefixed.

You are right, I had the same worries with previous task but I ended up accepting this. Surely worth reconsidering. I'll see in the final review.